### PR TITLE
Feature/order socket

### PR DESCRIPTION
**Pull request by ay0o5h**

### CODE REVIEW

File: service_restaurant/src/test/kotlin/org/thechance/service_restaurant/domain/validation/OrderValidationTest.kt [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+        val executable = Executable {
+            orderValidation.validateUpdateOrder(orderId = orderId, status = status)
+        }
+
+        val error = assertThrows(MultiErrorException::class.java, executable)
+        assertEquals(true, error.errorCodes.contains(INVALID_STATUS))
+    }
+
+    @Test
+    fun `should do nothing when orderStatus is valid`() {
```
Comment (Asiasama710) [inline service_restaurant/src/test/kotlin/org/thechance/service_restaurant/domain/validation/OrderValidationTest.kt:87]: u can name it like  (should pass) not (should do nothing)

File: service_restaurant/src/test/kotlin/org/thechance/service_restaurant/domain/validation/OrderValidationTest.kt [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+        val executable = Executable {
+            orderValidation.validateUpdateOrder(orderId = orderId, status = status)
+        }
+
+        val error = assertThrows(MultiErrorException::class.java, executable)
+        assertEquals(true, error.errorCodes.contains(INVALID_ID))
+    }
+
+    @Test
+    fun `should should throw MultiErrorException contains INVALID_ID when orderId is Invalid in getOrderById`() {
```
Comment (Asiasama710) [inline service_restaurant/src/test/kotlin/org/thechance/service_restaurant/domain/validation/OrderValidationTest.kt:33]: there is double should 😆

File: service_restaurant/src/main/kotlin/org/thechance/service_restaurant/api/endpoints/OrderRoutes.kt [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+                    restaurantId = restaurantId,
+                    manageOrder = manageOrder
+                )
+            }
+        }
+
+    }
+}
+
+private suspend fun broadcast(
```
Comment (Asiasama710) [inline service_restaurant/src/main/kotlin/org/thechance/service_restaurant/api/endpoints/OrderRoutes.kt:96]: i think this should be in different file !

Comment (ahmednasserzaza) [reply service_restaurant/src/main/kotlin/org/thechance/service_restaurant/api/endpoints/OrderRoutes.kt]: Done ✔

**Review by devsadeq — APPROVED** [review]

**Review by HaidyAbuGom3a — APPROVED** [review]

**Pull request merged**